#CompanyMessage.msg

#founder name as string
std_msgs/String founder 

#integer
int32 age_of_company
int32 number_of_employee
